Tasks/Gradle/Gradle.ps1

```diff
@@ -170,7 +170,7 @@ if($publishJUnitResultsFromAntBuild)
 	$runTitleMemberExists = CmdletHasMember "RunTitle"
 	if($runTitleMemberExists)
 	{
-		Publish-TestResults -TestRunner "JUnit" -TestResultsFiles $matchingTestResultsFiles -Context $distributedTaskContext -RunTitle $testRunTitle
+		Publish-TestResults -TestRunner "JUnit" -TestResultsFiles $matchingTestResultsFiles -Context $distributedTaskContext -RunTitle $testRunTitle -MergeResults $true
```

Comment:
check if this effects our e2e tests. make sure you run e2e tests before checkin.